doc/api/dgram.md

```diff
@@ -425,9 +425,7 @@ s.bind(1234, () => {
 });
 ```
 
-## `dgram` module functions
```

Comment:
No this will have them listed at the same level as the classes. It's the same as `http`, `https`, `readline`, `fs`, etc. L1 heading is the module name, L2 is each class and each static (if that's the right word) method and property. L3 is for class instance methods/props.

`dgram` was the odd one out, this was just for consistency. Although, having just gone to check I didn't do something dumb, I notice that `crypto` also pushes all the module functions/props down to L3, but there is seems to make sense.